Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1498,6 +1498,10 @@ - (void)URLSession:(NSURLSession *)session webSocketTask:(NSURLSessionWebSocketT
 
     cookieStorage.get()._overrideSessionCookieAcceptPolicy = YES;
 
+ALLOW_DEPRECATED_DECLARATIONS_BEGIN
```

Comment:
How about the configuration in `statelessSessionWithoutRedirects()`, in PrivateClickMeasurementNetworkLoaderCocoa.mm?

Reply:
It might be safe to enforce TLS 1.2 for PCM.

Reply:
I could update that, but do we actually need to support TLS 1.0/1.1 for private click measurement? It's an optional feature that does not affect page loads. The overlap between websites refusing to update their legacy server to TLS 1.2 and websites using PCM is likely 0. We could argue that removing legacy TLS improves privacy.

Reply:
It appears that we relied at the exact same time :)